manifests/minimize_access.pp

```diff
@@ -91,5 +92,13 @@
     password => '*',
   }
 
+  # i do not know how this works on anything else than debian/ubuntu
+  if ::operatingsystem == 'debian' or ::operatingsystem == 'ubuntu' {
```

Comment:
Did you mean `$::operatingsystem` here? And why not integrate this in the `case` on line 24ff?

Reply:
Indeed I do.
Not integrated with the case as I feel that case is used to set some of the variables. Although I'm not entirely sure why I'm using an `if` statement here, a `case` would be much better and easier to extend.